src/v3/src/components/Widget/index.tsx

```diff
@@ -286,7 +286,8 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
     >
       {/* Note that we need two theme providers until we fully migrate to odyssey-mui */}
       <MuiThemeProvider theme={mapMuiThemeFromBrand(brandColors)}>
-        <ScopedCssBaseline>
+        {/* the style is to allow the widget to inherit the parent's bg color */}
+        <ScopedCssBaseline sx={{ backgroundColor: 'inherit' }}>
```

Comment:
Is this related to this ticket?

Reply:
Oops, no, I was working with Lee on something and forgot to remove this 😅 will remove (this should be added in his branch for the ticket he is working on).